patch-webpack.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const f = 'node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js';
+
+fs.readFile(f, 'utf8', function (err,data) {
```

Comment:
Otherwise `ng build --prod` fails, see [here](https://travis-ci.org/Aam-Digital/ndb-core/builds/584295792?utm_source=github_status&utm_medium=notification). Let me know if you know a cleaner way to circumvent this.